.vscode/launch.json

```diff
@@ -28,13 +28,17 @@
         "--require",
         "ts-node/register",
         "-u",
-        "tdd",
+        "bdd",
```

Comment:
could you tell why do we need to change it?

Reply:
Yes I forgot to mention it.

I had this issue: https://stackoverflow.com/a/55761459/8357826

Can you reproduce it on your pc?